modules/network/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "vpc_id" {
+  value = aws_vpc.main.id
```

Comment:
Should we also output the subnetIDs so that we can assign resources to the correct subnet (private/public)?